android/modules/ui/src/java/ti/modules/titanium/ui/TableViewProxy.java

```diff
@@ -190,6 +190,7 @@ private void appendRowInternal(Object rows, KrollDict animation, boolean interna
 					row.getProperties().getString(TiC.PROPERTY_FOOTER)));
 
 			// Add row to section.
+			row.setParent(section);
```

Comment:
@m1ga This is already done by `section.add(row)` in next call. I am not sure what it would fix :).

Reply:
Please run the example code and you can see that it didn't use the event listeners when you reasign data to the table. `section.add(row)` is adding the row to the section, The PR is setting the parent to the row again. It looks like this is needed to get the event listener from the table again